shopinvader_delivery_carrier/services/delivery.py

```diff
@@ -159,4 +159,7 @@ def _get_base_search_domain(self):
         sale_obj = self.env[sale_service._expose_model]
         sale_domain = sale_service._get_base_search_domain()
         pickings = sale_obj.search(sale_domain).mapped("picking_ids")
```

Comment:
```suggestion
        sale_domain.append(('picking_type_id.code', '=', 'outgoing'))
        pickings = sale_obj.search(sale_domain).mapped("picking_ids")
```

Reply:
`sale_domain` is used to search on` sale.order`, not on `stock.picking`. So your suggested change won't work at all.

Reply:
ouch you are right :sweat: 
Anyway, why don't we search for pickings straight?
The domain would be:

```py
('picking_type_id.code', '=', 'outgoing'),
("sale_id.partner_id", "=", self.partner.id),
("sale_id.shopinvader_backend_id", "=", self.shopinvader_backend.id),
("sale_id.typology", "=", "sale"),
```

Reply:
IMO it's better to use the `picking_ids` field (I think this field is computed). So it avoid code duplication and if an integrator customize the `picking_ids` field, it doesn't have to update the shopinvader part.
Also, I think search using `.` are not optimized so it could take time (I don't know if it's less or more compared to `sale_obj.search(sale_domain).mapped("picking_ids")`).